Dockerfile

```diff
@@ -1,6 +1,9 @@
+ARG GIT_POINTER=master
+ARG PYTHON_VERSION=3.6
 
-FROM python:3.6
+FROM python:${PYTHON_VERSION}
+MAINTAINER tyler@prefect.io
```

Comment:
Make this `help@prefect.io` similar to `setup.py`

Reply:
fixed--LABEL maintainer="help@prefect.io"